src/mongocrypt-kms-ctx.c

```diff
@@ -1485,9 +1685,18 @@ _mongocrypt_kms_ctx_init_kmip_activate (mongocrypt_kms_ctx_t *kms_ctx,
    size_t reqlen;
    const uint8_t *reqdata;
 
+   if (!kms_ctx) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.